src/codegen/flags.rs

```diff
@@ -74,6 +74,17 @@ fn generate_flags(env: &Env, w: &mut Write, flags: &Bitfield, config: &GObject)
             "    #[must_use]"
         ));
     }
+    for derive in &config.derives {
+        let derive_str = match &derive.cfg_condition {
+            Some(condition) => format!(
+                "#[cfg_attr(any({}, feature = \"dox\"), derive({}))]",
+                condition,
+                derive.name
+            ),
+            None => format!("#[derive({})]", derive.name),
+        };
+        try!(writeln!(w, "    {}", derive_str));
+    }
```

Comment:
This is duplicate, so please extract it to `codegen\general.rs` with usage `tabs(indent)` instead prefix spaces